index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <title>Medicore </title>
+    <link href="./styles.css" rel="stylesheet">
+</head>
+<body>
+    <div class="header">
+        <div class="menu-bar">
+            <div class="menu-bar__logo menu-bar__logo--text-big">
+                <div class="menu-bar__link">Medical Core</div>
+            </div>
+            <div class="menu-bar__links menu-bar__links--text-muted">
+                <div class="menu-bar__link">Home</div>
+                <div class="menu-bar__link">Product</div>
+                <div class="menu-bar__link">Pricing</div>
+                <div class="menu-bar__link">Contact</div>
+            </div>
+            <div class="menu-bar__section">
+                <button class="button button--no-background">Login</button>
+                <button class="button button--uppercase button--background">join us <img src="./images/Vector.png"> </button>
+            </div>
+        </div>
+        <div class="welcome">
+            <div class="welcome__text">
+                <label class="welcome__desc welcome__text--primary welcome__text--big-in-sm">Welcome</label>
+                <label class="welcome__desc welcome__text--big">A Great Place to Receive Care</label>
+                <label class="welcome__desc welcome__text--muted">Get your best looking smile now!</label>
+                <div class="welcome__buttons">
+                    <button class="button button--background">Join Us</button>
+                    <button class="button button--outline">Learn More</button>
+                </div>
+            </div>
+            <div class="welcome__empty"></div>
+        </div>
+    </div>
+    <div class="content">
+        <div class="content__images">
+            
+            <img class="content__image" src="./images/lady-doctor.png">
+        </div>
+        <div class="content__text">
+            <div>
+                <hr class="content__separator">
+                <label class="content__desc content__text--big">Meet our experts</label>
+                <div class="content__desc content__text--muted">
+                    Problems trying to resolve the conflict between 
+                    the two major realms of Classical physics:
+                    Newtonian mechanics 
+                    <div class="welcome__desc welcome__text--primary welcome__text--padded">
```

Comment:
Here we have `welcome__text--primary`, I expect this to be inside a BLOCK called `welcome` but it is nowhere to be found as a parent here.

Reply:
I made it to reusable block